src/main/java/com/netgrif/application/engine/export/service/ExportService.java

```diff
@@ -137,10 +138,11 @@ public OutputStream fillCsvCaseData(List<CaseSearchRequest> requests, File outFi
         return buildCaseCsv(exportCases, config, outFile);
     }
 
-    protected OutputStream buildCaseCsv(List<Case> exportCases, ExportDataConfig config, File outFile) throws FileNotFoundException {
+    @Override
+    public OutputStream buildCaseCsv(List<Case> exportCases, ExportDataConfig config, File outFile) throws FileNotFoundException {
         Set<String> csvHeader = config == null ? buildDefaultCsvCaseHeader(exportCases) : config.getDataToExport();
         OutputStream outStream = new FileOutputStream(outFile, false);
-        PrintWriter writer = new PrintWriter(outStream, true);
+        PrintWriter writer = new PrintWriter(outStream, true, StandardCharsets.UTF_8);
```

Comment:
Charset should be a part of the `ExportDataConfig` class with default value of UTF 8

Reply:
Done